packages/backend/package.json

```diff
@@ -1,19 +1,27 @@
 {
   "name": "example-backend",
   "version": "0.1.1-alpha.12",
-  "main": "dist/index.cjs.js",
+  "main": "src/index.ts",
   "types": "src/index.ts",
   "private": true,
   "license": "Apache-2.0",
   "engines": {
     "node": ">=12"
   },
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.cjs.js",
+    "types": "dist/index.d.ts"
+  },
   "scripts": {
     "build": "backstage-cli backend:build",
+    "build-image": "backstage-cli backend:build-image example-backend",
```

Comment:
Doesn't pick up the local package automatically?

Reply:
it's the image tag 😅

Reply:
slapped a `:latest` on there to clarify